Argo/Functions/decode.swift

```diff
@@ -13,3 +15,21 @@ public func decode<T: Decodable where T == T.DecodedType>(object: AnyObject) ->
 public func decode<T: Decodable where T == T.DecodedType>(object: AnyObject) -> Decoded<[T]> {
   return Array<T>.decode(JSON.parse(object))
 }
+
+// MARK: decode with a root key
+
+public func decode<T: Decodable where T == T.DecodedType>(rootKey: String, _ object: AnyObject) -> T? {
```

Comment:
I could possibly change the name so it reads better: `decodeWithRootKey` instead.